ui/app/AppLayouts/Wallet/controls/ManageTokensDelegate.qml

```diff
@@ -111,6 +111,11 @@ DropArea {
                     else
                         root.controller.showHideRegularToken(symbol, flag)
                     root.controller.saveSettings()
+                    if (!flag) {
```

Comment:
Moved here to prevent a double toast when hiding from the main screen

Reply:
I don't understand why this double toast happens if we listen to the hidden signal in the controller from the store. This seems to indicate we are triggering twice the signal, right?

Reply:
Ah, but I see the rest of toasts being called, as I was expecting, inside the store :thinking: